<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerCropVolumeModule</class>
 <widget class="qSlicerWidget" name="qSlicerCropVolumeModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Parameters</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="InputVolumeLabel">
          <property name="text">
           <string>Input volume</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="InputVolumeComboBox">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLScalarVolumeNode</string>
           </stringlist>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="InputROILabel">
          <property name="text">
           <string>Input ROI</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="InputROIComboBox">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLAnnotationROINode</string>
           </stringlist>
          </property>
          <property name="editEnabled">
           <bool>true</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="VisibilityButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../Libs/qMRMLWidgets/Resources/qMRMLWidgets.qrc">
            <normaloff>:/Icons/VisibleOff.png</normaloff>
            <normalon>:/Icons/VisibleOn.png</normalon>:/Icons/VisibleOff.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="ROIVisibilityLabel">
          <property name="text">
           <string>ROI visibility</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Interpolation mode</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="IsotropicCheckbox">
             <property name="text">
              <string>isotropic output voxel</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="NNRadioButton">
               <property name="text">
                <string>Nearest Neighbor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LinearRadioButton">
               <property name="text">
                <string>Linear</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="WSRadioButton">
               <property name="text">
                <string>WindowedSinc</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="BSRadioButton">
               <property name="text">
                <string>B-spline</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CropButton">
        <property name="text">
         <string>Crop!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>273</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Libs/qMRMLWidgets/Resources/qMRMLWidgets.qrc"/>
  <include location="../qSlicerCropVolumeModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerCropVolumeModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputVolumeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerCropVolumeModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputROIComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
